### PR TITLE
openorienteering-mapper: 0.8.1 -> 0.8.1.2 and fix templates

### PR DESCRIPTION
**Pull request by mpickering**
###### Motivation for this change

Version bump and fix an issue with reading template files.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/gis/openorienteering-mapper/default.nix [hunk 1]
```diff
@@ -47,8 +48,10 @@ stdenv.mkDerivation rec {
     stdenv.lib.optionalString stdenv.isDarwin ''
     # Fixes "This application failed to start because it could not find or load the Qt
     # platform plugin "cocoa"."
+    echo ${qtimageformats}/${qtbase.qtPluginPrefix}
```
Comment (jtojnar) [inline pkgs/applications/gis/openorienteering-mapper/default.nix]: Why are you printing it?

Comment (mpickering) [reply pkgs/applications/gis/openorienteering-mapper/default.nix]: Woops, removed that now. Thanks.

**Comment by boomshroom** [comment]
Builds on NixOS, but attempting to run complains about an incompatible QT library. That's probably why it was marked Mac-only.

**Comment by jtojnar** [comment]
The QT error is recurring issue, not anything to do with this PR. See for example https://github.com/NixOS/nixpkgs/issues/17448

**Comment by mpickering** [comment]
The package should work fine with NixOS but I haven't tested it hence why the attribute is not set.

**Pull request merged**